modules/installation-user-infra-machines-advanced-customizing-live-network-config.adoc

```diff
@@ -8,6 +8,11 @@
 = Modifying a live install {boot-media} with customized network settings
 You can embed a NetworkManager keyfile into the live {boot-media} and pass it through to the installed system with the `--network-keyfile` flag of the `customize` subcommand.
 
+[WARNING]
+====
+When creating a connection profile, you must use a `.nmconnection` filename extension in the connection profile's filename. If you do not use a `.nmconnection` filename extension, the cluster will apply the connection profile to the live environment, but it will not apply the configuration when the cluster first boots up the nodes, resulting in a setup that does not work.
```

Comment:
Nitpick: consider replacing "in the connection profile's filename" with "in the filename of the connection profile."

Reply:
Alternatively, reword to "append the `.nmconnection` extension to the filename of the connection profile" or similar.

Reply:
Done.